Intoduce el new_password: aHt37gh8
Acceso autorizado
Acceso autorizado
Traceback (most recent call last):
  File "C:\Users\Lenovo\Desktop\pythoncore2025\class_5\Section_3Error Detection\Lab3\Section_1 Programming Practice\task_4.py", line 11, in <module>
    if passwod != 'aHt37gh8':
       ^^^^^^^
NameError: name 'passwod' is not defined. Did you mean: 'password'?